### ISSUE TITLE
Previous Collaborative Environment

### ISSUE THREAD

**Issue by jan-janssen** [mbruns91/graphathon#8]
As reference to the previous collaborative pyiron environments:
https://github.com/pyiron/collaborative_pyiron_in_binder

**Comment by samwaseda** [comment]
Yeah but that’s a binder feature that can be put on top of what we have here. I don’t think we need to put them together.

**Comment by jan-janssen** [comment]
It is primarily to keep everybody up to date about which related developments we already have.
